### ISSUE TITLE
Docs on "Record-based APIs: the nested records case" not in main branch

### ISSUE THREAD

**Issue by tfc** [haskell-servant/servant#1701]
Hello,

i stumbled upon this multiple times:

1. I need to know how to use record based nested apis, so i go to https://docs.servant.dev (what else?)
2. It's not there. So i google.
3. Google leads me to https://docs.servant.dev/en/inserting_doc_namedroutes/cookbook/namedRoutes/NamedRoutes.html

I think this should be merged and part of the normal main docs, or are there reasons against?

**Comment by ysangkok** [comment]
What makes you think that this is related to git branches? I don't actually know where this is stored or built from, so this is not rhetorical.

**Comment by tfc** [comment]
I guessed. There is a branch that's called exactly like that version of the docs.

**Comment by tchoutri** [comment]
Yeah ReadTheDocs does this. We have to bring https://github.com/haskell-servant/servant/pull/1542 to the finish line

**Comment by tchoutri** [comment]
Fixed by https://github.com/haskell-servant/servant/pull/1726
